File: GraphGenerator/fixtures/execute.rb
require 'fileutils'

path = Dir.pwd
strings = path.split("/")

dotFilename = "#{strings.last}.dot"
svgFilename = "#{strings.last}.svg"

system("dot -Tsvg #{dotFilename} -o #{svgFilename}")
system("dot -Tsvg generated.dot -o generated.svg")

File: GraphGenerator/fixtures/execute.py
import os

path = os.getcwd()
strings = path.split("\\")

dir = strings.pop()

dotFilename = dir + ".dot"
svgFilename = dir + ".svg"

os.system("dot -Tsvg %s -o %s" % (dotFilename, svgFilename))
os.system("dot -Tsvg generated.dot -o generated.svg")